tests/cql_query_test.cc

```diff
@@ -4261,3 +4261,135 @@ SEASTAR_TEST_CASE(test_rf_expand) {
         });
     });
 }
+
+
+
+/*
+ * The following tests are for schema describe command
+ */
+
+static void cmp_schema(schema_ptr s1, schema_ptr s2) {
+    BOOST_CHECK_EQUAL(s1->ks_name(), s2->ks_name());
+    BOOST_CHECK_EQUAL(s1->cf_name(), s2->cf_name());
```

Comment:
One more really easy check to add would be to check if the schemas have the same column names (and perhaps types). There are multiple ways of listing column names from the schema, one of them is `columns_by_name()`, which gives you a map where keys are ordered column names - easy to compare.

I'm also fine with leaving it as a todo for the future.